### PR TITLE
LG-11817: RSpec and CI configuration added from IDP

### PR DESCRIPTION
**Pull request by olatifflexion**
## 🎫 Ticket

[Link to the relevant ticket.](https://cm-jira.usa.gov/browse/LG-12034)

## 🛠 Summary of changes

 RSpec and CI configuration added from IDP

### CODE REVIEW

File: config/initializers/01_redis.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+REDIS_POOL = ConnectionPool.new(size: IdentityConfig.store.redis_pool_size) do
```
Comment (zachmargolis) [inline config/initializers/01_redis.rb:1]: what do we use redis for in this app? do we need it?

Comment (olatifflexion) [reply config/initializers/01_redis.rb:1]: i think we will need it later so i added that.

Comment (zachmargolis) [reply config/initializers/01_redis.rb:1]: I think we don't have any use cases defined yet, so I'd opt to remove it for now, it's very easy to add back in later if needed

Comment (olatifflexion) [reply config/initializers/01_redis.rb:1]: sure!

**Review by zachmargolis — APPROVED** [review]

**Pull request merged**